packages/editor/src/components/page-attributes/README.md

```diff
@@ -0,0 +1,160 @@
+Here components that provide the functionality available in `Page Attributes` panel of the document sidebar are documented.
+Using these components, one can add a UI that allows changing page attributes in different scenarios/needs.
+
+PageAttributesCheck
+===================
```

Comment:
I believe these should be H2s — we should only have a single H1 per page.

Reply:
We should also standardise on a single type of heading syntax: `#` or `=`. I prefer `#` but this isn't a blocker, just an observation. (I might look for markdown linters today and write a proposal for including one.)